src/gasman.c

```diff
@@ -2043,7 +2094,7 @@ UInt CollectBags (
             /* free the identifier                                         */
             *(Bag*)(header->link) = FreeMptrBags;
             FreeMptrBags = header->link;
-
+            
```

Comment:
?

Reply:
fixed

Reply:
Still is in the diff? But anyway, not important